src/components/PurchaseOrder/CloseOrder/CloseOrderModal.js

```diff
@@ -35,104 +34,85 @@ const DEFAULT_REASONS = {
 
 class CloseOrderModal extends Component {
   static propTypes = {
-    orderId: PropTypes.string,
-    workflowStatus: PropTypes.string,
+    orderNumber: PropTypes.string,
     closingReasons: PropTypes.arrayOf(PropTypes.object),
     closeOrderSubmit: PropTypes.func.isRequired,
+    isOpen: PropTypes.bool.isRequired,
+    onCancel: PropTypes.func.isRequired,
   }
 
   constructor(props) {
     super(props);
 
     this.state = {
-      openModal: false,
       reason: '',
       note: '',
     };
   }
 
-  openModal = () => {
-    this.setState({
-      openModal: true,
-    });
+  reasonChanged = (e) => {
+    this.setState({ reason: e.target.value });
   }
 
-  closeModal = () => {
-    this.setState({
-      openModal: false,
-    });
+  noteChanged = (e) => {
+    this.setState({ note: e.target.value });
   }
 
   render() {
-    const { orderId, workflowStatus, closingReasons, closeOrderSubmit } = this.props;
-    const isVisible = orderId && workflowStatus === WORKFLOW_STATUS.open;
+    const { orderNumber, closingReasons, closeOrderSubmit, onCancel, isOpen } = this.props;
```

Comment:
`closeOrderSubmit` - sounds weird, it might be more clear to name callbacks like `closeOrder`, `cancel`, etc.